source/plugin/text/representations/index.rst

```diff
@@ -3,41 +3,40 @@ Text Serializers
 ================
 
 .. javadoc-import::
-    org.spongepowered.api.text.Text
-    org.spongepowered.api.text.serializer.TextParseException
-    org.spongepowered.api.text.serializer.TextSerializer
-    org.spongepowered.api.text.serializer.TextSerializers
+    net.kyori.adventure.text.Component
+    net.kyori.adventure.text.serializer.ComponentSerializer
+    net.kyori.adventure.text.serializer.gson.GsonComponentSerializer
+    net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer
+    net.kyori.adventure.text.serializer.plain.PlainComponentSerializer
+    net.kyori.adventure.serializer.configurate4.ConfigurateComponentSerializer
     java.lang.String
 
-:javadoc:`TextSerializer`\ s provide a convenient way to serialize and de-serialize :javadoc:`Text` instances. There
-are three applicable formats:
+..
+	Note to editors: These used to be many other files in this folder, which were made irrelevant. This file was not moved to keep the URLs to this page alive.
```

Comment:
These -> There. Also please add a new line to keep <120 characters.
I think we need to find a better way to handle this. Many of the API-7 docs were made *redundant*.
Do we need to seek and destroy references to the page, or keep it as a placeholder? Input wanted.

Reply:
My thoughts were about external links to this page. Internally that would be fine, find every reference and edit them.